transportable-udfs-trino/src/main/java/com/linkedin/transport/trino/StdUdfWrapper.java

```diff
@@ -110,14 +144,17 @@ public ScalarFunctionImplementation specialize(BoundVariables boundVariables, in
         - (new Random()).nextInt(initialJitterInt));
     boolean[] nullableArguments = stdUDF.getAndCheckNullableArguments();
 
-    return new ScalarFunctionImplementation(true, getNullConventionForArguments(nullableArguments),
-        getMethodHandle(stdUDF, metadata, boundVariables, nullableArguments, requiredFilesNextRefreshTime));
+    return new ChoicesScalarFunctionImplementation(
+        functionBinding,
+        FAIL_ON_NULL,
```

Comment:
If arguments of a Transport UDF are defined as non-nullable and if a null value is passed to it, the current behaviour is that the UDF will return a null. In Trino, previously we didn't have any special handling on Transport side for this as Trino behaviour was also the same. Does this change that behaviour to fail instead if nulls are passed?

Reply:
updated it to NULLABLE_RETURN.

Reply:
I am surprised there wasn't a case in the `examples` project to catch this. Would it be possible to add a test for this?

I think its fine if it is not addressed as part of this PR, but atleast we should make sure we aren't causing a regression here.

Reply:
> If arguments of a Transport UDF are defined as non-nullable and if a null value is passed to it, the current behaviour is that the UDF will return a null. 

We already [have tests ](https://github.com/linkedin/transport/blob/master/transportable-udfs-examples/transportable-udfs-example-udfs/src/test/java/com/linkedin/transport/examples/TestArrayElementAtFunction.java#L33)to validate the above behaviour but it looks like this field doesn't govern that behaviour.

This parameter (NULLABLE_RETURN) and the subsequent one `getNullConventionForArguments` only define the convention and used to validate if the function can be adapted to this pattern.